components/event/EventPrizeContainer.js

```diff
@@ -0,0 +1,196 @@
+import { useState } from 'react';
+import { Blue } from '@components/input/Button';
+import { POST } from '@apis/defaultApi';
+import { ICON_COUPON } from '@utils/constants/icons';
+import styled from '@emotion/styled';
+import Heading from '@components/input/Heading';
+import { getDate } from '@utils/functions';
+import { NO_IMAGE } from '@utils/constants/images';
+export default function EventPrizeContainer({
+  eventId,
+  eventPrizes,
+  eventType,
+  eventStatus,
+  applyStatus,
+  headers,
+}) {
+  const [applySts, setApplySts] = useState(applyStatus);
+  const [status, setStatus] = useState(eventStatus || 0);
+
+  function applyEvent(eventId, eventPrizeId, prizeName) {
+    if (confirm('경품' + prizeName + '에 응모하시겠습니까?')) {
+      POST(
+        '/event/participant',
+        {
+          eventId,
+          eventPrizeId,
+        },
+        headers,
+      ).then((res) => {
+        if (res) {
+          if (res.success) {
+            alert('응모 성공');
+            setApplySts('disabled');
+          } else {
+            alert('이미 응모 하셨습니다.');
+          }
+        } else {
+          alert('잠시후 다시 시도해주세요');
+        }
+      });
+    }
+  }
+
+  function ProductCard({ prize }) {
+    return (
+      <>
+        <div className="h-100 place-content-center justify-center content-center">
+          <ImgSection>
+            <img
+              className="prize-img w-48 h-48 object-center"
+              src={prize.productImg || NO_IMAGE}
+            />
+          </ImgSection>
+          <h4 className="text-black text-2xl text-center font-bold leading-snug tracking-tight mb-3">
+            {prize.productName || '상품명'}
+          </h4>
+          {eventStatus === 1 && (
+            <div className="flex justify-center">
+              <Blue
+                buttonText={'이벤트 응모'}
+                onClickFunc={() =>
+                  applyEvent(eventId, prize.eventPrizeId, prize.productName)
+                }
+                attr={{ disabled: applySts }}
+                css={{
+                  marginTop: 'auto',
+                  marginBottom: '10px',
+                }}
+              />
+            </div>
+          )}
+        </div>
+      </>
```

Comment:
맨 바깥쪽 <> </>는 제거해도 될것같아요

```suggestion
        <div className="h-100 place-content-center justify-center content-center">
          <ImgSection>
            <img
              className="prize-img w-48 h-48 object-center"
              src={prize.productImg || NO_IMAGE}
            />
          </ImgSection>
          <h4 className="text-black text-2xl text-center font-bold leading-snug tracking-tight mb-3">
            {prize.productName || '상품명'}
          </h4>
          {eventStatus === 1 && (
            <div className="flex justify-center">
              <Blue
                buttonText={'이벤트 응모'}
                onClickFunc={() =>
                  applyEvent(eventId, prize.eventPrizeId, prize.productName)
                }
                attr={{ disabled: applySts }}
                css={{
                  marginTop: 'auto',
                  marginBottom: '10px',
                }}
              />
            </div>
          )}
        </div>
```